mmeval/metrics/rouge.py

```diff
@@ -0,0 +1,307 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# This class is modified from `torchmetrics
+# <https://github.com/Lightning-AI/metrics/blob/master/src/torchmetrics/text/rouge.py>`_.
+import re
+from collections import Counter
+from difflib import SequenceMatcher
+from typing import (TYPE_CHECKING, Any, Callable, Dict, List, Optional,
+                    Sequence, Tuple, Union)
+
+from mmeval import BaseMetric
+from mmeval.metrics.utils import get_tokenizer, inference_language
+from mmeval.utils import try_import
+
+if TYPE_CHECKING:
+    import nltk
+else:
+    nltk = try_import('nltk')
+
+
+def _get_results(matches: int, pred_len: int,
+                 reference_len: int) -> Dict[str, float]:
+    """This computes precision, recall and F1 score based on matches.
+
+    Args:
+        matches (int): A number of matches or a length of
+            the longest common subsequence.
+        pred_len (int): A length of a tokenized predicted sentence.
+        reference_len (int): A length of a tokenized referenced sentence.
```

Comment:
```suggestion
        reference_len (int): A length of a tokenized referenced sentence.

```

Reply:
It seems that there is no change between the modified code and the original code?